### DISCUSSION TITLE
DAI 2024-2025 - Practical work 1 - Guillaume Fragnière and Killian Viquerat

### DISCUSSION THREAD

**Discussion by GFragniere** [heig-vd-dai-course/heig-vd-dai-course#326] [Show and tell]
Our CLI application will be able to do those tasks:
- Turn a colored image into a gray-scale image.
- Reduce pepper noise on an image.
- Take a colored image, turn it to gray scale and turn it into an ASCII text file.
- Blur an image. (optionnal)

**Comment by GeraudSilvestri** [comment]
That seems ok for us, you're good to go

**Comment by Killian-Viquerat** [comment]
Here is our submit for the practical work:

https://github.com/Killian-Viquerat/DAI_Practical_Work_1_Magic_Image_CLI/tree/v1.0.0

Commit Hash: aee3b19f039d6b8f6c4183f7cedc801471ce0a9a
